Documentation/helm-operator.md

```diff
@@ -74,19 +55,12 @@ helm repo add rook-master https://charts.rook.io/master
 helm search repo rook-ceph --versions
```

Comment:
I can't comment on that line (54), but let's remove the `# For Helm V3.x` comments / lines.

Reply:
done